.github/workflows/breaking-change-alert.yaml

```
@@ -66,7 +66,11 @@ jobs:
           PR_AUTHOR: ${{ inputs.pr_author }}
         run: |
           escape_json() {
-            echo "$1" | jq -R -c . | grep -E '^".*"$' | awk '{print substr($0, 2, length($0) - 2)}' && [ "${PIPESTATUS[2]}" -eq 0 ]
+            local flag="-R"
+            if [ "$2" = "stream" ]; then
+                flag="-sR"
+            fi
+            echo "$1" | jq ${flag} -c . | grep -E '^".*"$' | awk '{print substr($0, 2, length($0) - 2)}' && [ "${PIPESTATUS[2]}" -eq 0 ]
```

I see no reason to differentiate between a single-line string and a multi-line string. The behavior is the same either way for a single-line string, and is an improvement for a multi-line string. Why not just treat it as multi-line unconditionally?

```suggestion
            echo "$1" | jq -sR -c . | grep -E '^".*"$' | awk '{print substr($0, 2, length($0) - 2)}' && [ "${PIPESTATUS[2]}" -eq 0 ]
```

---

@KyleFromNVIDIA left a comment [here](https://github.com/rapidsai/shared-workflows/pull/260#issuecomment-2460242280).